Jasmine/Common/ViewControllers/PhrasesExplorerViewController.swift

```diff
@@ -3,29 +3,34 @@ import SnapKit
 
 class PhrasesExplorerViewController: UIViewController {
 
-    @IBOutlet private weak var phrasesTableView: UIView!
+    @IBOutlet private var explorerNavigationItem: UINavigationItem!
+    @IBOutlet private var phrasesTableView: UIView!
     fileprivate var phrasesTable: PhrasesTableViewController!
-    fileprivate(set) var viewModel: PhrasesExplorerViewModel!
+    fileprivate var viewModel: PhrasesExplorerViewModel!
     private var searchController: UISearchController!
-    private var isMarkable: Bool!
 
-    /// Callback to execute when the explorer view is dismissed, 
-    /// passes the selected phrases to the callback function
-    private var onDismiss: ((Set<Phrase>) -> Void)?
+    /// Callback to execute when the save button is pressed. 
+    /// Passes the selected phrases to the callback function
+    private var onSaveCallBack: ((Set<Phrase>) -> Void)?
```

Comment:
For `hideSaveButtonIfSaveNotSet`? That wouldn't work because `onSaveCallBack` is set before `viewDidLoad` so `explorerNavigationItem` is still nil and will cause a crash on access